common/lib/xmodule/xmodule/tests/test_randomize_module.py

```diff
@@ -97,8 +97,8 @@ def inner_get_module(descriptor):
         randomize_module = RandomizeModule(
             randomize_descriptor,
             self.system,
-            DictFieldData({}),
-            ScopeIds(None, None, self.course.id, self.course.id)
+            field_data=None,
```

Comment:
why do you even need to set `field_data` here? If you leave it blank, tests pass as well